test/powershell/Language/Parser/TypeAccelerator.Tests.ps1

```diff
@@ -410,15 +410,20 @@ Describe "Type accelerators" -Tags "CI" {
                     Accelerator = 'ordered'
                     Type        = [System.Collections.Specialized.OrderedDictionary]
                 }
+                @
```

Comment:
Looks like this type is causing tests to fail.

Reply:
My bad. Fixed.